show/main.py

```diff
@@ -1683,14 +1700,7 @@ def temperature():
     """Show device temperature information"""
     cmd = 'tempershow'
     run_command(cmd)
-
```

Comment:
Can you please add this back too? I think this firmware section was also accidently removed ?

Reply:
I think I have already added it back ,is that any mistake?

Reply:
![image](https://user-images.githubusercontent.com/39321413/77532195-57e2fa00-6ecf-11ea-8002-54250201cf52.png)